src/components/Pager/index.tsx

```diff
@@ -0,0 +1,314 @@
+import React from 'react';
```

Comment:
Please refer to this updated file
https://github.com/toggle-corp/toggle-ui/blob/dfb69ebe734e7a90cbd2424c2abc9c2c85848561/src/components/Pager/index.tsx